### DISCUSSION TITLE
Logo license

### DISCUSSION THREAD

**Discussion by LilScottyPippen** [community/community#50850] [Repositories] [answered]
### Select Topic Area

Question

### Body

1. In my project, I am using the GitHub logo as a link to GitHub OAuth and to the repository with the project. Do I need any permissions to use the GitHub logo
2. I use the project logo in my project. Do I need to add a logo license to the repository

**Comment by radar-blip** [comment, marked as answer]
Here's a little info that should help you with that: https://github.com/logos

**Reply by LilScottyPippen** [reply]
What about my logos? Do I need any licenses or is MIT enough?

**Reply by radar-blip** [reply]
I don't think you need to do anything else to use your own logo, of course adding a MIT license cant hurt.

**Reply by LilScottyPippen** [reply]
Thanks

**Reply by radar-blip** [reply]
Anytime. 👍🏻
